### PR TITLE
[T4A6][W11-B4]Ong Heng Le 

### PR DESCRIPTION
**Pull request by initialshl**

### CODE REVIEW

**Comment by initialshl** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:4]: no \* imports

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+import java.util.ArrayList;
+import java.util.HashSet;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsAnyNull() {
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:13]: method name does not follow coding style.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        
+        // initialize different data types
+        Object n = null;
+        Object o = new Object();
+        String s = "";
+        int i = 0;
+        int[] a = { 0, 0, 0, 0, 0 };
+        
+        // no input
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:22]: refactor the tests. one test per method. basically each method is a test case.

**Comment by cmkumar87** [comment]
@initialshl review comments and close PR.

**Pull request closed without merging**